documentation/writing_tasks.md

```diff
@@ -439,9 +439,10 @@ error. Bolt merges the 'file' and 'line' keys with results unless 'file' is pres
 
 ### Returning sensitive data
 
-To return secrets from a task, use the `_sensitive` key in the output. Bolt
-will treat the result as sensitive and won't allow it to be printed to the
-console or log.
+To return secrets from a task, use the `_sensitive` key at the top level of the output. Bolt will
+treat the result as sensitive and won't allow it to be printed to the console or log. Note that this
+will not obfuscate data under `_sensitive` if `_sensitive` is not at the top level, for example `{
+user: _sensitive: { carmen: sandiego } }` will not be made sensitive.
```

Comment:
```suggestion
does not obfuscate data under `_sensitive` if `_sensitive` is not at the top level. 
For example, Bolt does not treat this `user` as sensitive: 
`{user: _sensitive: { carmen: sandiego } }`.
```
Could we also provide the "correct" example here?

Reply:
The correct example is directly below, I can make that more clear!